python/tvm/relay/transform/quantization/quantized_operators/dense.py

```diff
@@ -0,0 +1,183 @@
+from typing import *
+
+import numpy as np
+import tvm
+from tvm import relay
+from tvm.relay.dataflow_pattern import (
+    DFPatternCallback,
+    _DFPatternCallback,
+    is_constant,
+    is_op,
+    wildcard,
+)
+from tvm.relay.op import nn, tensor
+from tvm.relay.transform.quantization import calibration_rewrite_utils
+
+QParams = NamedTuple(
```

Comment:
I like the use of NamedTuple here, I've never seen it before!

Reply:
Yup it's great! Now I just wish I could get VSCode to at least try to type check against it :/